html/changelogs/Intichaplain.yml

```diff
@@ -0,0 +1,4 @@
+author: Intigracy
+delete-after: True
+changes: 
+- rscdel: "Removes the message that everyone sees when the chaplain joins (which I added). Sounded good on paper, not so well in execution."
```

Comment:
get with the program

in-description changelogs are the hot new meme, it allows maintainers to shitpost in people's Changelogs/force changelogs to be added/remove any and all attempts for people to say "but changelogs r 2 hard!!1!"

ask MrStonedOne or your local coderbus congressman how to set it up